### PR TITLE
creates models for organizations & projects

### PR DESCRIPTION
**Pull request by george-silva**
This pull request is the first one to target explansion.

This is a simple pull request. All it does is add two new models:

1. Organization (has owner and name);
2. Projects (has organization)

Also this pull request creates two important things:

1. An admin user for planscape
2. 4 Projects to represent our current Projects

### CODE REVIEW

**Review by lastminutediorama — APPROVED** [review]
looks good to me, I read through it twice and couldn't find anything to complain about 😆

File: src/planscape/projects/models.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        help_text="Visibility choice to control project visibility to external clients",
+    )
+
+    capabilities = ArrayField(
+        models.CharField(
+            choices=ProjectCapabilities.choices,
+            max_length=64,
+        ),
+        default=default_capabilities,
+        help_text="Models our per project feature flags",
```
Comment (pflopez) [inline src/planscape/projects/models.py:84]: Not sure if I understand this comment

File: src/planscape/projects/migrations/0002_auto_20240425_1845.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+from pathlib import Path
+from django.db import migrations
+from django.conf import settings
+from projects.models import ProjectVisibility
+from django.contrib.gis.geos import GEOSGeometry, MultiPolygon
+
+
+def get_geometry(region_name):
+
+    filename = f"projects/migrations/{region_name}.geojson"
```
Comment (pflopez) [inline src/planscape/projects/migrations/0002_auto_20240425_1845.py:11]: super nit here, and maybe the regions geojson should live somewhere else (fixtures?) given that the files are not migrations themselves?

File: src/planscape/core/models.py [hunk 3]
```diff
@@ -15,8 +17,40 @@ class Meta:
...
+    dead_or_alive = models.Manager()
+
+    deleted_at = models.DateTimeField(
+        verbose_name="Deleted at",
+        help_text="Define if the entity has been deleted or not and when",
+        null=True,
+    )
+
+    def delete(self, *args, **kwargs):
+        """Soft delete an entity."""
```
Comment (pflopez) [inline src/planscape/core/models.py:36]: Maybe this comment belongs to  the `else` below not when checking for `hard_delete`

**Review by pflopez — APPROVED** [review]
looks great! just a couple of comments but not at all blockers

**Pull request merged**